service/controller/key/key.go

```diff
@@ -13,6 +13,8 @@ const (
 	// defaultDNSLastOctet is the last octect for the DNS service IP, the first
 	// 3 octets come from the cluster IP range.
 	defaultDNSLastOctet = 10
+	// capiNameLabel is the upstream Cluster API cluster name label.
+	capiNameLabel = "cluster.x-k8s.io/cluster-name"
```

Comment:
Could we import it from `capi`? I see the project already has it as a dependency.

Reply:
You are correct - fixed.